tasks.py

```diff
@@ -0,0 +1,214 @@
+"""Tasks for use with Invoke."""
```

Comment:
we can get ansible-test sanity to ignore the errors on this file by creating a `tests/sanity/ignore-2.10.txt` file and adding these lines to it
```
tasks.py future-import-boilerplate
tasks.py metaclass-boilerplate
```

https://docs.ansible.com/ansible/latest/dev_guide/testing/sanity/ignores.html